examples/koa/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "_example-koa",
+  "private": true,
+  "version": "0.1.7",
+  "dependencies": {
+    "@reframe/react-kit": "0.1.7",
+    "react": "^16.3.2",
+    "koa": "^2.5.1",
+    "koa-router": "^7.4.0",
+    "koa-compose": "^4.0.0"
+  },
+  "checkDeps": {
+    "skip": true
+  }
+}
```

Comment:
So far the plugin examples live within the plugin directory. E.g. `plugins/react-router/example/`. To keep things consistent let's do the same for this plugin and let's move the example to `plugins/koa/example/`

Reply:
Done

Reply:
You moved it to `examples/koa`. Let's move it to `plugins/koa/example` instead

Reply:
Done